### PR TITLE
Add initial XCUITests

### PR DESCRIPTION
**Pull request by isabelrios**
This is a PR to try to add a basic XCUITest to the app and hopefully set the ground for more :)

### CODE REVIEW

**Comment by isabelrios** [comment]
Hi @CelesteTang ! I'm trying to write some XCUITests for the app, and even though they passed on BuddyBuild, the build failed with this error:
```
  » note: Planning build
    ✗ Provisioning profile "buddybuild Adhoc - org.mozilla.ios.FirefoxVPN.network-extension" doesn't support the group.org.mozilla.ios.Guardian App Group. (in target 'WireGuardTunnel' from project 'FirefoxPrivateNetworkVPN')
    ✗ Provisioning profile "buddybuild Adhoc - org.mozilla.ios.FirefoxVPN.network-extension" doesn't match the entitlements file's value for the com.apple.security.application-groups entitlement. (in target 'WireGuardTunnel' from project 'FirefoxPrivateNetworkVPN')
    ** ARCHIVE FAILED **
```

Am I missing something? thanks for your help!

**Comment by CelesteTang** [comment]
@isabelrios it seems like Provisioning profile was changed somehow. We might need Release team's help.

**Comment by isabelrios** [comment]
> @isabelrios it seems like Provisioning profile was changed somehow. We might need Release team's help.

Release team help...@JohanLorenzo I heard your name around this project ;) not sure if you are the person to ask or if you could send me to the correct one to investigate this...

**Comment by JohanLorenzo** [comment]
Hey! 

I started to look into this issue with the help of @mozbhearsum. First of all, we realized the profile`buddybuild Adhoc - org.mozilla.ios.FirefoxVPN.network-extension` was shown as invalid on https://developer.apple.com/account/resources/profiles/list. For some reason it was pointing to the macos package name. I fixed it.

Although, that didn't solve the issue. Per https://stackoverflow.com/questions/41867538/provisioning-profile-doesnt-include-the-com-apple-security-application-groups-e, it seems there's something to do on XCode. I checked that the app group exists, so that part looks good. 

What you all think?

**Comment by isabelrios** [comment]
Thanks @JohanLorenzo I'm not very familiar with that but I will take a look... @CelesteTang do you know how that was set before the error? or any idea about what to change? Thanks!

File: FirefoxPrivateNetworkVPNUITests/FirefoxPrivateNetworkVPNUITests.swift
Comment (CelesteTang) [inline FirefoxPrivateNetworkVPNUITests/FirefoxPrivateNetworkVPNUITests.swift:49]: This assertion should be removed if it's not been used anymore. Same as line 55 and 60.

**Comment by CelesteTang** [comment]
> Hey!
> 
> I started to look into this issue with the help of @mozbhearsum. First of all, we realized the profile`buddybuild Adhoc - org.mozilla.ios.FirefoxVPN.network-extension` was shown as invalid on https://developer.apple.com/account/resources/profiles/list. For some reason it was pointing to the macos package name. I fixed it.
> 
> Although, that didn't solve the issue. Per https://stackoverflow.com/questions/41867538/provisioning-profile-doesnt-include-the-com-apple-security-application-groups-e, it seems there's something to do on XCode. I checked that the app group exists, so that part looks good.
> 
> What you all think?

@isabelrios @JohanLorenzo Someone seems to uncheck the app group configuration for `org.mozilla.ios.FirefoxVPN.network-extension`. I fixed the issue and rebuilt it successfully. Now we could move on.

**Comment by isabelrios** [comment]
Thanks @CelesteTang I uncommented those lines, please let me know if you have more feedback or any other comments, thanks!

**Comment by isabelrios** [comment]
@AaronMT builds are green on BB now, tagging you in case you want to add some comments (sorry I can't add you or anyone as reviewer...)

**Review by CelesteTang — APPROVED** [review]
Look good to me!

**Comment by isabelrios** [comment]
@CelesteTang sorry for bothering you again but who should I contact to be able to merge this or to ask for another review if needed? how is the process to get a PR landed? I don't have rights on this repo even to ask for review...Thanks!

**Comment by CelesteTang** [comment]
@isabelrios you've gotten my approval so I could merge this PR if there is no another reviewer you want to ask for. Let me know if you want me to merge the code or add another reviewer. Thank you!

File: FirefoxPrivateNetworkVPNUITests/BaseTestCase.swift
Comment (AaronMT) [inline FirefoxPrivateNetworkVPNUITests/BaseTestCase.swift:9]: 2020?

**Review by AaronMT — APPROVED** [review]

File: FirefoxPrivateNetworkVPNUITests/BaseTestCase.swift
Comment (isabelrios) [reply FirefoxPrivateNetworkVPNUITests/BaseTestCase.swift:9]: mm not sure... I added it as it was defined in the existing file [FirefoxPrivateNetworkVPNUITests.swift ](https://github.com/mozilla-mobile/guardian-vpn-ios/blob/develop/FirefoxPrivateNetworkVPNUITests/FirefoxPrivateNetworkVPNUITests.swift)

**Pull request merged**
